src/components/index.js

```diff
@@ -1,6 +1,7 @@
 export * from './map';
 export * from './screens';
 
+export * from './bookmarks';
```

Comment:
here i think was my thought to structure the folders first separated by a line break from the files, that are directly in /components. so the bookmarks line should go up, right?

Reply:
you are right. changed it in 8a84e9e25ce56257a4c54ca67a967bcc32fcc310